litert/c/litert_tensor_buffer.h

```diff
@@ -168,6 +168,10 @@ LiteRtStatus LiteRtGetTensorBufferOpenClMemory(LiteRtTensorBuffer tensor_buffer,
                                                cl_mem* cl_mem_addr);
 #endif  // LITERT_HAS_OPENCL_SUPPORT
 
+// Return an error if the backing buffer is not a OpenVINO remote tensor.
+LiteRtStatus LiteRtGetTensorBufferOpenVINOTensorBuffer(
```

Comment:
Let's not limit this for OpenVINO.

LiteRtGetTensorBufferCustomTensorBufferHandle